src/test/regress/expected/multi_sequence_default.out

```diff
@@ -541,7 +541,9 @@ SELECT start_metadata_sync_to_node('localhost', :worker_1_port);
 
 -- Check that various ALTER SEQUENCE commands
 -- are not allowed for a distributed sequence for now
-CREATE SEQUENCE seq_8;
+-- Also check that various sequence options are passed on to the worker
+-- correctly
+CREATE SEQUENCE seq_8 INCREMENT BY 3 CACHE 10 CYCLE;
```

Comment:
I think it'd be nice to have a test which explicitly specifies the type

Reply:
Done